common/views/pages/_document.tsx

```diff
@@ -64,6 +64,7 @@ class WecoDoc extends Document {
     return (
       <Html lang="en" className="is-keyboard">
         <Head>
+          <meta charSet="utf-8" />
```

Comment:
Maybe worth a comment explaining this needs to fall in the first 1024 bytes, so we don't break it again?